cluster/examples/kubernetes/ceph/nfs-test.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: rook-ceph # namespace:cluster
 spec:
   rados:
-    # RADOS pool where NFS client recovery data is stored.
+    # RADOS pool where NFS client recovery data is stored. Don't use EC pool, it doesn't support omap which is required by ganesha.
```

Comment:
Actually how about removing the comments from this nfs-test.yaml? The -test.yaml files are intended for test scenarios, then to avoid duplication we can keep the documentation in the nfs.yaml.

Reply:
All the comments or just this one ?

Reply:
That's fine for now to just not update it, but in a future PR I'd suggest even removing all the comments in nfs-test.yaml unless they are specific to that yaml.